### DISCUSSION TITLE
关于原神5.0前瞻，一位玩家致大家的信

### DISCUSSION THREAD

**Discussion by HFOxin** [share121/inter-knot#2348] [其他]
原神5.0前瞻即将开始，或许有人对米哈游已经很是不满，但是我想说，作为一个正常玩家，我呼吁，还我们一个良好的社区氛围，请各位不要公然在弹幕区里辱骂玩家。
我们一致认为，互联网不应该让一些别有用心的人以此传播自己的负面情绪，或者说，不应该在任何地方随意传播恶意情绪。

在原神4.8危机与7.22事变之后，截止到8月16日9：10，我已经在bilibili被人骂了一个月，我想说，任何人都不应该随意辱骂他人，何况用的还是“原批”“米孝子”如此下流，下贱，无耻的污秽之词。

自从原神开服以来，社区的节奏从未停息，我们是不是快忘记了，“原批”就是一个侮辱人的词汇，而“米孝子”更是将此进行延伸，将侮辱的含义不断扩大，上升到了做人的基本价值与祖宗亲戚。

结合现在的评论区大环境，身为一个原神与绝区零玩家，我再次呼吁大家：米哈游可能令你不满，但玩家无罪。我们不能成为网络暴力的受害者。

我希望大家能够在5.0前瞻直播间里，为维护社区环境出一把力。

感谢各位。

**Comment by shenguojia123** [comment]
实际上那些真蛰虫的口器都被剪了，直播间一点动静都没有

**Comment by GanlinSama** [comment]
楼上，要真是这样那我整整半个小时是在举报谁的逆天发言？

**Reply by shenguojia123** [reply]
抱歉哈我当时没看直播，听人说没啥动静

**Reply by soil-of-forset** [reply]
看直播的时候直接把弹幕关了，眼不见，心不烦。

**Reply by 00zzxx** [reply]
> 抱歉哈我当时没看直播，听人说没啥动静

这个是有治安官上台之后才没啥动静，治安官一走立马又有了<img src="https://bbs-static.miyoushe.com/static/2023/11/16/44d02375dbff4fde5ea3eadc82730131_8648553945620661617.png" alt="厨师邦布-疲惫" width="75" >

**Comment by WRYYY-S** [comment]
今天直接错过了魔神战争没看到。

我觉得罢，目前可以正常化的社区估计就这里了，毕竟是民间的，而且搭建在github上，有一定访问难度，xxs和水军不容易涌入，在这发就不是5毛一条了

别的社区罢，你难道指望xxs和机器人听你的劝吗？很蓝的辣。

别的不说，能够保护正常玩家讨论的目前只有绳网，只有这种有门槛且独裁（褒义）的社区才能保护正常玩家的讨论。

放宽心罢，等相关法律真正落实了再期待有所变化。

**Reply by 00zzxx** [reply]
确实，赞同，相比其他平台要来这里还是比较麻烦的
